### PR TITLE
[MIG] product_supplierinfo_for_customer: Migration to 15.0

### PR DESCRIPTION
**Pull request by nalejhandro**

### CODE REVIEW

**Comment by nalejhandro** [comment]
Hi @pedrobaeza  can you help me with a review of this migration?

**Comment by pedrobaeza** [comment]
Sorry, not working on this right now.

File: product_supplierinfo_for_customer/models/product_product.py [hunk 1]
```diff
@@ -131,6 +131,7 @@ def _select_customerinfo(
         params.update({"partner_id": partner.id})
         domain = self._prepare_domain_customerinfo(params)
         res = self.env["product.customerinfo"].search(
-            domain, order="product_id, sequence", limit=1
-        )
-        return res
+            domain
+        ).sorted(lambda s: (s.sequence, s.min_qty, s.price, s.id))
```
Comment (rousseldenis) [inline product_supplierinfo_for_customer/models/product_product.py]: @nalejhandro Why changing the order from previous version ?

EDIT: Ok, saw the commit message. Could you maybe put that change in a separate commit ?

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review

**Review by rousseldenis — APPROVED** [review]

**Review by hugho-ad — APPROVED** [review]

**Comment by CarmenMiranda** [comment]
Hi, @moylop260 @luisg123v Could your take a look at this?

This was already approved twice 😃

**Pull request merged**